src/tools/compiletest/src/common.rs

```diff
@@ -275,6 +276,10 @@ pub struct Config {
     /// not vice versa.
     pub target_panic: PanicStrategy,
 
+    /// A map from target cfg keys to their values. If the key is present with no value,
+    /// the vector will contain the empty string.
+    pub target_cfg: HashMap<String, Vec<String>>,
```

Comment:
I think it'd be useful to give a short example here (e.g. the has atomic kv pairs).